### ISSUE TITLE
Error while creating a new user or updating password in V1.6

### ISSUE THREAD

**Issue by lawvisiondev** [2amigos/yii2-usuario#508]
### What steps will reproduce the problem?

After the update to v1.6 creating a new user or updating a password using the webinterface causes 500 internal server errors.
**Create User**: (using new valid e-mail-address, new unique username, password
Open new form - /index.php?r=user/admin/create 
After entering the username, password the console shows an 500 internal server error (jquery.js)

Console shows error: 
try {
// Do send the request (this may raise an exception)
_xhr.send( options.hasContent && options.data || null );_
} catch ( e ) { ... }

Log: 
2023-02-24 21:08:09 [IP-ADDRESS][3][63kvbvmu2shmsb4aqsl05pb0jv][error][yii\base\UnknownPropertyException] yii\base\UnknownPropertyException: Getting unknown property: Da\User\Model\User::auth_tf_type in /var/www/html/WEBSITE/htdocs/advanced/vendor/yiisoft/yii2/base/Component.php:154

In Database field auth_tf_type is available (no value)

**Update User.**
Same problem when updating a password/username of an existing user. (index.php?r=user/admin/update&id=5)
Updating Profile-Data (f.ex. index.php?r=user/admin/update-profile&id=5) is working without problems

### What is the expected result?

Create: New User is created
Update: Updated Data (Username, Password, e-mail) is saved.

### What do you get instead?

 500 internal server error 
Getting unknown property: Da\User\Model\User::auth_tf_type

**Comment by maxxer** [comment]
Did you override the user model? Can you post module config?

**Comment by lawvisiondev** [comment]
No, there is no override for the user model. 
fyi: We use the yii2 advanced template with frontend and backend - usuario works with no problems until the update to v1.6

Here is my module config:

    'modules' => [        
		'user' => [
            'class' => Da\User\Module::class,
            "administratorPermissionName" => 'admin',
            'enableGdprCompliance' => true,
            'enableRegistration' => false,
            'enableTwoFactorAuthentication' => false,
		], ...
 
Thank you!

**Comment by maxxer** [comment]
how long ago did you run the migration? could it be you still have schema cached to the old version? can you try from cli

```
./yii cache/flush-schema
```

**Comment by lawvisiondev** [comment]
You are right. Now it works - m000000_000010_add_auth_tf_type_auth_tf_mobile_phone_columns_to_user_table.php caused the problem. Thank you!
